omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/DrugOrderSubclassHandler1_10.java

```diff
@@ -34,12 +34,14 @@
 import org.openmrs.module.webservices.rest.web.resource.impl.NeedsPaging;
 import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.DrugOrderSubclassHandler1_8;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.OrderResource1_8;
 
 /**
  * Exposes the {@link org.openmrs.DrugOrder} subclass as a type in
  * {@link org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_10.DrugOrderSubclassHandler1_10}
  */
-@SubClassHandler(supportedClass = DrugOrder.class, supportedOpenmrsVersions = { "1.10.*", "1.11.*", "1.12.*" })
+@SubClassHandler(parent = OrderResource1_8.class, path = "drugorder", supportedClass = DrugOrder.class, supportedOpenmrsVersions = {
```

Comment:
Did you make this change intentionally?

Reply:
Yes sir

Reply:
How is it related to the ticket?

Reply:
Code couldn't compile without it since I modified the `@Subclass` annotation class. I have just put a comment on it down there.

Reply:
Is path also required to get rid of the compiler error?

Reply:
Yes sir